experimental/PropertyDDS/packages/property-inspector-table/package.json

```diff
@@ -21,10 +21,11 @@
   ],
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint",
-    "build:compile": "concurrently npm:tsc npm:build:esnext npm:webpack:svgs",
+    "build:compile": "concurrently npm:tsc npm:build:esnext npm:build:webpack",
```

Comment:
Make sure you update the reference to this script name in `partial-checkout/package.json` and property-inspector/package.json` in the `experiemental/ProeprtyDDS/examples`